src/app/v1/views/users.py

```diff
@@ -38,7 +38,30 @@ def get(self, user_id: int):
 			raise InternalServerError
 
 		return user
+	
+@api.route('/<int:user_id>/plants')
+class UserPlants(Resource):
+	@marshal_list_with(ListPlantSchema)
+	def get(self, user_id: int):
+		try:
+			plants = get_plants(user_id, db.session)
+		except Exception as e:
+			logging.error('failed to get plants')
+			logging.exception(e)
+			raise InternalServerError
+		
+		return plants
+
+	@serialize_with(NewPlantSchema)
+	def post(self, user_id: int, body: Plant):

```

Comment:
Doesn't follow the schema. We need a multipart upload so users can upload images.